io.github.congard.qnvsm.yml

```diff
@@ -0,0 +1,22 @@
+app-id: io.github.congard.qnvsm
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: qnvsm
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland?

Reply:
It should, as far as Qt supports Wayland

Reply:
then
```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```